src/browsing.md

```diff
@@ -234,8 +234,8 @@ they can be found in a submenu of the context menu.
-| Forget         | Move currently selected cards to the end of the new queue. The existing review history is preserved. In 2.1.50+, there are options to restore the original card position, and to reset the card's lapse and repetition counters.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| Set Due Date   | Turns cards into review cards if they are new, and makes them due on a certain date. This can be useful for moving cards forward or back a few days when your study schedule is interrupted. Entering a range like "60-90" will make the selected cards due between 60 and 90 days from now. New cards will have their interval set to the same delay, but review cards will be rescheduled without changing their current interval, unless an exclamation mark ("!") is included at the end of the range. Note that the answer time is not recorded when manually scheduling cards, since the action can be performed even outside of review, and Anki isn’t aware of which card may or may not be shown at the time.                                                                                                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
Similar to the discussion on the searching page, you're changing backticks to double quotes here. Personally, I think backticks/kbd are more legible than double quotes, especially for things like example input. Thoughts?